packages/webapp/src/routes/land/check-legal-agreement-details.js

```diff
@@ -20,6 +21,11 @@ const handlers = {
     }, {
       inProgressUrl: constants.routes.CHECK_LEGAL_AGREEMENT_DETAILS
     })
+
+    const registrationTaskStatus = getIndividualTaskStatus(request.yar, REGISTRATIONCONSTANTS.LEGAL_AGREEMENT)
+    if (registrationTaskStatus !== 'COMPLETED') {
+      return h.redirect(constants.routes.REGISTER_LAND_TASK_LIST)
+    }
```

Comment:
looks like we do this check only in this case to check status, why is that?

Reply:
discussed